docker-build.sh

```diff
@@ -82,6 +82,41 @@ buildOpenJDKViaDocker()
   # shellcheck disable=SC1090
   source "${BUILD_CONFIG[DOCKER_FILE_PATH]}/dockerConfiguration.sh"
 
+    local openjdk_core_version=${BUILD_CONFIG[OPENJDK_CORE_VERSION]}
```

Comment:
Typically we try to keep the configuration building inside `configureBuild.sh` so that the config is set in stone when the build is initiated. This block should probably be in `configureBuild.sh`

Reply:
actually this block is already in `configureBuild.sh`, is there an issue with it being there?

Reply:
The block in `configureBuild.sh` currently configures the variables JRE/JDK_PATH for native builds only and does not reconfigure these variables (or create new variables) when building inside a docker container, hence the cause of the bug. Therefore the solution I implemented involved copying the code block from `configureBuild.sh` and adding to `docker-build.sh`.

Reply:
@johnoliver Any further comment after Haroon's statement above?